### DISCUSSION TITLE
Website Server Block

### DISCUSSION THREAD

**Discussion by Spencer021** [gameserverapp/Platform#539] [Suggestions]
Instead of showing the map could we have it show the server name instead? This will help identify them better for servers with multiple maps. Even more with ASA since the only map is island!

**Comment by mvaessen** [comment]
Hi, i was just looking into your suggestion and noted that the game server name listed on the block is the `website name` listed on the screenshot here: https://docs.gameserverapp.com/dashboard/game_servers/getting_started#change-names

Have you considered changing the name for one of them so website visitors can also separate the two :)
